queries.sql

```diff
@@ -31,3 +23,28 @@ SELECT * FROM animals not name != 'Gabumon';
 
 /*return all data of Pikachu and Devimon*/
 SELECT * FROM animals WHERE weight_kg >=  10.4 AND weight_kg <= 17.3;
+
+-- How many animals are there?
+-- How many animals have never tried to escape?
+-- What is the average weight of animals?
+-- Who escapes the most, neutered or not neutered animals?
+-- What is the minimum and maximum weight of each type of animal?
+-- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
+-- Remember to add these queries to your queries.sql file.
+
+/*Return the total number of animals in the database*/
+SELECT COUNT(*) FROM animals;
+
+/*counts how many animals have tried to escape*/
+SELECT COUNT(*) FROM animals WHERE escape_attempts > 0;
```

Comment:
We have a tiny problem with these question 👀 We should be answering the number of animals that **never** tried to escape rather than how many have. 

`How many animals have never tried to escape?`

Maybe try something like:

    SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;

Kindly make the necessary changes to your query so that we meet the requirements 🚀